Calculator/Calculator/Model/ExtenssionPrimitiveDataType/String.swift

```diff
@@ -9,4 +9,28 @@ extension String {
     func split(with target: Character) -> [String] {
         return self.split(separator: target).map { String($0) }
     }
+    
+    var isZero: Bool {
+        return self == "0"
+    }
+    
+    var isPrime: Bool {
+        return self.contains(".")
+    }
+    
+    var convertSign: String {
+        if self.hasPrefix("-") {
+            return String(self.dropFirst())
+        } else {
+            return "-\(self)"
+        }
+    }
```

Comment:
네이밍 상 `mutating` 메서드로 작성되는 것이 더 적절할 것 같습니다.

```suggestion
    mutating func convertSign() {
        if hasPrefix("-") {
            self = String(dropFirst())
        } else {
            self = "-" + self
        }
    }
```

Reply:
`mutating func`로 수정했습니다!